Corona-Warn-App/src/main/java/de/rki/coronawarnapp/http/WebRequestBuilder.kt

```diff
@@ -20,11 +20,8 @@
 package de.rki.coronawarnapp.http
 
 import KeyExportFormat
-import com.google.protobuf.InvalidProtocolBufferException
-import de.rki.coronawarnapp.exception.ApplicationConfigurationCorruptException
-import de.rki.coronawarnapp.exception.ApplicationConfigurationInvalidException
```

Comment:
These Exceptions should not be out just because of your Typo Fix, right?

Reply:
True, my bad. Don't know what happened there. Is fixed now :)